MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm

```diff
@@ -449,6 +449,8 @@
 void MVKCommandEncoder::endCurrentMetalEncoding() {
 	endMetalRenderEncoding();
 
+	_computePipelineState.markDirty();
```

Comment:
Did you investigate whether the other two state trackers in `MVKCommandEncoder::finalizeDispatchState()` (`_computeResourcesState` and `_computePushConstants`) need to be marked dirty too?

Reply:
No, I didn't. It seems likely that they do, though honestly I'm not familiar with the code enough to be sure. This made sense and works for my case.

Reply:
Okay. Can you add them here anyway, please?

Reply:
Sure, I pushed changed commits